include/Dialect/CGGI/Transforms/Passes.td

```diff
@@ -0,0 +1,19 @@
+#ifndef INCLUDE_DIALECT_CGGI_TRANSFORMS_PASSES_TD_
+#define INCLUDE_DIALECT_CGGI_TRANSFORMS_PASSES_TD_
+
+include "mlir/Pass/PassBase.td"
+
+def SetDefaultParameters : Pass<"cggi-set-default-parameters"> {
+  let summary = "Set default parameters for CGGI ops";
+  let description = [{
+  This pass adds default parameters to all CGGI ops as `cggi_params` named
```

Comment:
nit: it would be nice to mention the default parameters somewhere in the docs for here and the LWE pass, either inline or with a link to the source code (it seems like the jaxite params are used)

Reply:
Done!